scripts/globals/mobskills/Bubble_Shower.lua

```diff
@@ -9,7 +9,25 @@ require("scripts/globals/monstertpmoves");
 
 ---------------------------------------------------
 
-function onMobSkillCheck(target,mob,skill)
+function onMobSkillCheck(target,mob,skill,zone)
+	mob = mob:getID();
+	if (mob == 17129519) then
+	local mobids = {17129520,   -- King Arthro's Knights

```

Comment:
same formatting probs I pointed out in your other PR, just so you know. And you need to rebase this off newer DSP, you are missing the onMobDeath script changes. `killer:` is replaced by `ally:` in most cases